compiler/src/test-integration/java/io/neow3j/compiler/ArraysConverterIntegrationTest.java

```diff
@@ -132,6 +167,23 @@ public static String getEmptyStringFromCreatedArray() {
         public static Object[] createObjectArray() {
             return new Object[]{1, "hello, world!", io.neow3j.devpack.Hash160.zero()};
         }
+
+        public static int variableLengthParam(int a, int... integers) {
+            int sum = integers[0];
+            for (int i : integers) {
+                sum += i;
+            }
+            return sum + a;
+        }
+
+        static Event1Arg<io.neow3j.devpack.Hash160> event;
+
+        public static void callRuntimeNotifyWithVarArgs(io.neow3j.devpack.Hash160... hashes) {
```

Comment:
I know it's a bit of an overkill, but can we make a test with an `Object` (complex type) that is not from the Runtime/devpack? (i.e., any complex type, like `String`, or `Integer`)

If this doesn't make sense, then it's fine and we can move on.